### DISCUSSION TITLE
Remote Student - Struggling to verify enrollment

### DISCUSSION THREAD

**Discussion by itsjustriley** [community/community#137870] [GitHub Education]
### Hi everyone,

I'm applying for the student developer pack, but have been rejected a couple times. 

> The school you selected does not appear to have a campus location in your country.

> The image you selected does not appear to contain your school name.

> You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote.

* I'm on the east coast of Canada, but a remote student at a university on the west coast of the United States
* I do not have a student card as a remote student
* I do not have any physical documents to take pictures of because all my learning is online
* The entire program is online so there is no special indicator that *I* specifically am a remote student

I tried submitting screenshots (via taking a photo of my screen, because theres's no upload file option) of my schedule with name, student number, school name, and student number on it (but perhaps it was too blurry)? I also submitted a photo of the program website indicating that the program is fully taught online. I am not able to obtain physical documents to verify my enrolment, and there is no indication on any of the digital documents I have access to (schedule, unofficial transcript, etc) that indicate I specifically am remote as the entire program is remote.  I also don't have access to a printer. 

What is the best way for me to verify I am a student? I could probably request a customized enrolment letter be mailed across the continent so I can photograph it, but am hoping there's a workaround.

**Comment by ajgalipeau** [comment]
I am trying the same thing, and also running into this issue. It requires us to use a photo when I could simply submit documentation from the school's canvas/application center. I'm a graduate student in computer science and currently unable to **Verify Enrollment**. Adding my comment to this question for _additional visibility_.

**Reply by tfidelman** [reply]
I am having the same issue. I am only given the option to upload a photo via my webcam, when I could easily submit documentation to resolve this. Hoping for a quick resolution.

**Reply by ajgalipeau** [reply]
Hi @tfidelman ,
I was able to resolve this. When I took a picture with the webcam, I made sure to take as clean of a picture as possible. If you are not near campus and need to explain, really and truly explain where you are, what semester you are in, the classes you are taking, and let your transcript show some elements of this for verification. As far as the ID card, Merrimack in my case is all digital now. We don't have a year on it, so it forced me to overcommunicate school information regarding my status. Best of luck!
